streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -324,6 +326,17 @@ object CheckpointReader extends Logging {
     }
     None
   }
+
+  /**
+   * If using the keytab to login user in security mode, this checkpoint application has to login
+   * before reading files from hdfs.
```

Comment:
I will look at this today.

On Monday, July 6, 2015, andrewor14 <notifications@github.com> wrote:

> In streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala
> <https://github.com/apache/spark/pull/7158#discussion_r33969975>:
>
> > @@ -324,6 +326,17 @@ object CheckpointReader extends Logging {
> >      }
> >      None
> >    }
> > +
> > +  /**
> > +   * If using the keytab to login user in security mode, this checkpoint application has to login
> > +   * before reading files from hdfs.
>
> maybe add that this is only for YARN mode.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/7158/files#r33969975>.
>


-- 

Thanks,
Hari